packages/interbit-root-covenant/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "interbit-root-covenant",
+  "version": "0.4.43",
+  "description": "Root covenant, for handling automatic manifest updates",
+  "main": "src/index.js",
+  "scripts": {
+    "test": "mocha \"src/tests/**/*.test.js\""
+  },
+  "dependencies": {
+    "interbit-covenant-tools": "^0.4.43"
+  },
+  "devDependencies": {
+    "assert": "^1.4.1",
+    "mocha": "^4.0.1"
```

Comment:
We should standardize on jest

Reply:
#607 addresses that. When it's complete I will merge it back into cascading deploy via master